api-reference/user-app/abdm-connect/registration.yml

```diff
@@ -1855,6 +1860,12 @@ components:
         profile:
           $ref: '#/components/schemas/ProfileResponse'
           description: Profile of the user, will be present if SkipState is abha_end
+        refresh_token:
```

Comment:
It should have been part of many response objects like the mobile login/create and Aadhaar login/create

Reply:
Can you check if it's appearing on all these responses?